src/enable_variable_ops/ngraph_enter_in_catalog.h

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#ifndef NGRAPH_TF_ENTER_IN_CATALOG_H_
+#define NGRAPH_TF_ENTER_IN_CATALOG_H_
+#pragma once
+
+#include "tensorflow/core/graph/graph.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph_catalog.h"
+
+using namespace std;
+namespace ng = ngraph;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+// 1. Populate the input_variable_map
```

Comment:
Can we change this comment to describe the `what/why` of the events here, in addition to `how`. eg: `collect XYZ info by looking for ABC pattern in the tensorflow graph by populating input_variable_map`

Reply:
Done